tools/Makefile

```diff
@@ -0,0 +1,14 @@
+all:	directory-cache
+
+directory-cache: pip
+	@if [[ ! -f authority_certificate ]]; then \
+	  echo "Please copy authority_* files to this directory"; \
+	  false; \
+	fi
+	mkdir -f directory
+	./gen_fresh_dirinfo.py
+	mv consensus.txt microdescriptors.txt directory
+
+pip:
```

Comment:
This target is an implicit `PHONY` as the file `pip` is never generated, so the dependencies will always get reinstalled

Reply:
Yes, any idea how to change that? Add a 'phony' file that will be created upon installation of the dependencies?

Reply:
Yep, that's the way to do it :)

https://www.gnu.org/software/make/manual/html_node/Empty-Targets.html

Reply:
OK, done...